app/gallery/[slug]/page.tsx

```
@@ -8,6 +8,13 @@ interface ArtworkPageProps {
   params: Promise<{ slug: string }>
 }
 
+// Helper function to sanitize text for metadata
+function sanitizeText(text: string): string {
+  if (!text || typeof text !== 'string') return ''
+  // Remove control characters and other problematic characters
+  return text.replace(/[\x00-\x1F\x7F-\x9F]/g, '').trim()
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)